nerfstudio/data/pixel_samplers.py

```diff
@@ -278,7 +278,7 @@ def sample_method(
         mask: Optional[Tensor] = None,
         device: Union[torch.device, str] = "cpu",
     ) -> Int[Tensor, "batch_size 3"]:
-        if mask:
+        if isinstance(mask, torch.Tensor):
```

Comment:
nit: we import Tensor, so 
`if isinstance(mask, Tensor):`
or
`if mask is not None:`
would be preferred